nfs/exports.go

```diff
@@ -216,7 +216,7 @@ func restartNFSMountd() error {
 	exportsLock.Lock()
 	defer exportsLock.Unlock()
 	log.Infof("restarting nfs-mountd")
-	output, err := GetLocalExecutor().ExecuteCommand("chroot", "/noderoot", "container-systemctl", "restart", "nfs-mountd")
+	output, err := GetLocalExecutor().ExecuteCommand("chroot", "/noderoot", "ssh", "localhost", "systemctl", "restart", "nfs-mountd")
```

Comment:
Is this change required? Are we seeing issues with the current container-systemctl command that requires these changes?

Reply:
yes, there is no command named container-systemctl. And issuing systemctl commands need ssh prior to that.